modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -257,6 +260,14 @@ When forwarding logs to external services, additional metadata might be availabl
 | labels_type | Specifies the type of deployment that the pod is running. | MuleApplication
 |===
 
+== Send a Test Message
+
+You can send a test message to verify connectivity between Runtime Fabric and the third-party log forwarding solution.
+
+. In Runtime Manager, select *Runtime Fabrics* in the sidebar navigation.
+. Select the applicable Runtime Fabric based on the name used during installation.
+. Select the *Log Forwarding* tab.
+. Select ?
```

Comment:
@hegderohit89 Is this needs to be filled in by @paigehargrave ?
Here is the content:
. Select Forward application logs to an external service.
. Select a service and configure the required (and optional) fields for external logging service.
. *Apply changes* for saving and applying configuration to the Runtime Fabric cluster.
. To test the configuration and reachability of the external service, Trigger the test message under the selected service i.e. Configuration applied just now. Send a test message.
. Verify the test message from checking the verification text being forwarded to external logging solution successfully.